deepmd/pd/model/atomic_model/base_atomic_model.py

```
@@ -0,0 +1,580 @@
+    preset_out_bias : Dict[str, List[Optional[paddle.Tensor]]], optional
+        Specifying atomic energy contribution in vacuum. Given by key:value pairs.
+        The value is a list specifying the bias. the elements can be None or np.array of output shape.
+        For example: [None, [2.]] means type 0 is not set, type 1 is set to [2.]
+        The `set_davg_zero` key in the descrptor should be set.
+
+    """
+
+    def __init__(
+        self,
+        type_map: List[str],
+        atom_exclude_types: List[int] = [],
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)